library/func_misc.py

```diff
@@ -513,3 +513,19 @@ def Call(self, args):
             raise error.AssertionErr(msg, args.LeftParenToken())
 
         return value.Null
+
+
+class EvalExpr(vm._Callable):
+
+    def __init__(self, expr_ev):
+        # type: (expr_eval.ExprEvaluator) -> None
+        self.expr_ev = expr_ev
+
+    def Call(self, args):
+        # type: (typed_args.Reader) -> value_t
+        lazy = args.PosExpr()
+        args.Done()
+
+        result = self.expr_ev.EvalExpr(lazy, loc.Missing)
```

Comment:
Melvin just added args.LeftParenToken() or similar, so we can use that instead of `loc.Missing`

It will point to the `(` in `evalExpr()`